modules/ROOT/pages/gov-cloud-features.adoc

```diff
@@ -3,41 +3,42 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Government Cloud includes the full functionality of the runtime plane (services and Mule runtime engine) and a subset of the control plane (browser management) Anypoint Platform features.
-
-You must configure Mule to run in FIPS-compliant mode. See xref:mule-runtime::fips-140-2-compliance-support.adoc[FIPS 140-2 Compliance Support].
-
-Although it is not part of Government Cloud, you can use Anypoint Studio offline to build applications to deploy in Government Cloud. See xref:studio::set-credentials-in-studio-to.adoc[Configuring Anypoint Platform Credentials].
-
-[cols="1,2,2",grid="none",options="header"]
-.Features in Government Cloud
-|===
-||Included|Excluded
-3+^h|xref:design-center::index.adoc[Design Center]
-||xref:design-center::design-create-publish-api-specs.adoc[*API Designer*]|xref:design-center::about-designing-a-mule-application.adoc[*Flow designer*]
-||xref:design-center::design-create-publish-api-specs.adoc[*Visual API Designer*]|
-3+^h|xref:exchange::index.adoc[*Exchange*]
-||All|None
-3+^h|*Management Center*
-.10+||xref:access-management::index.adoc[*Access management*]|xref:monitoring::index.adoc[*Anypoint Monitoring*]
-|xref:object-store::osv2-guide.adoc[*Object Store*]|xref:anypoint-security::index-policies.adoc[*Edge Policies*]
-|xref:api-manager::latest-overview-concept.adoc[*API Manager*]|xref:anypoint-security::tokenization.adoc[*Tokenization*]
-|Runtime Manager: xref:runtime-manager::cloudhub.adoc[*CloudHub*]|xref:api-manager::analytics-landing-page.adoc[*Anypoint Analytics*]
-||xref:runtime-fabric::index.adoc[*Runtime Fabric*]
-||xref:mq::index.adoc[*Anypoint MQ*]
-||xref:visualizer::index.adoc[*Anypoint Visualizer*]
-||xref:data-gateway::index.adoc[*Data Gateway*]
-||xref:anypoint-security::index.adoc[*Secrets manager*]
-||xref:anypoint-security::index.adoc[*Anypoint Security*]
-|===
-
-// |xref:design-center::design-create-publish-api-specs.adoc[*Visual API designer*]|Flow designer
-// 2+^h|xref:exchange::index.adoc[*Exchange*]
-// 2+^h|xref:studio::index.adoc[*Anypoint Studio*]
-// |xref:access-management::index.adoc[*Access management*]|Anypoint Monitoring
-// .2+|xref:anypoint-security::index.adoc[*Anypoint Security*]|xref:anypoint-security/tokenization.adoc[Tokenization]
-// |xref:api-manager::latest-overview-concept.adoc[*API Manager*]|Anypoint Analytics
-// |Runtime Manager: xref:runtime-manager::cloudhub.adoc[*CloudHub*]|Runtime Fabric
+Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engines) functionality. Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
```

Comment:
According to the Style Ref. and the Term Presentation List, we should present the full product name at first use in a topic, and that is "MuleSoft Government Cloud."
Does this name continue to be debated? Because it should really be "Anypoint Platform Government Cloud," I hear.
In either case, it should not be abbreviated until spelled out once per topic.

Reply:
```suggestion
Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the functionality of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engine instances). Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
```
Should we add something about what this means for your audience? How does this fit it its journey?